### ISSUE TITLE
License the Project as GPLv2

### ISSUE THREAD

**Issue by biergaizi** [hardenedlinux/grsecurity-reproducible-build#1]
Currently the Project is copyrighted by the TYA Company. Since there are no any licenses in the repo, one must assume he/she couldn't even to use it and the project is "all rights reversed" by a company.

Being a fact that HardenedLinux community itself actually is **not** bonded to **any single** company or organization, we should put a `COPYING` file of GPLv2 into the repo, which is the license of the Linux and PaX/grsecurity.

**Comment by citypw** [comment]
Yes, HardenedLinux is not bonded to any commercial company. This project was funded by TYA in the initial phrase. And then the maintainence work has been done by our night job without any payment. IMOHO, is GPLv3 has any conflicts with copyright?

We released this project under GPLv3 in the 1st place:

https://github.com/hardenedlinux/grsecurity-reproducible-build/blob/master/LICENSE.txt

I added the License info in the README.
